Sentry.Xamarin.Forms/Extensions/SentryOptionsExtensions.cs

```diff
@@ -5,12 +5,12 @@ namespace Sentry
 {
     public static class SentryOptionsExtensions
     {
-        public static void DisableBreadcrumbForXamlWarnings(this SentryOptions option)
+        public static void DisableBreadcrumbForXamlWarnings(this SentryOptions _)
         {
-            SentryXamarinFormsIntegration.LogXamlErrors = false;
+            SentryXamarinFormsIntegration.Options.Value.XamarinLoggerEnabled = false;
```

Comment:
Ideally instead of relying on static state, you'd access the `options` you got as a parameter, find the integration from the list by its type, and toggle the value of it.